src/main/java/uk/gov/hmcts/ccd/data/user/DefaultUserRepository.java

```diff
@@ -155,7 +155,7 @@ public String getUserId() {
 
     private boolean filterRole(final String jurisdictionId, final String role) {
         return startsWithIgnoreCase(role, String.format(RELEVANT_ROLES, jurisdictionId))
-            || ArrayUtils.contains(AuthCheckerConfiguration.getCitizenRoles(), role);
+            || ArrayUtils.contains(AuthCheckerConfiguration.getCitizenRoles(), role) || applicationParams.getCcdRoleWhitelist().contains(role);
```

Comment:
is `ArrayUtils.contains(AuthCheckerConfiguration.getCitizenRoles(), role);` still required?
could we move "citizen", "letter-holder" to ccd.access-control.cross-jurisdictional-roles config property?

Reply:
actually, in terms of code understandability, it would be better to remove the hardcoded array, but introduce the following:
ccd.access-control.citizen-roles="citizen", "letter-holder"
and keep `ArrayUtils.contains(AuthCheckerConfiguration.getCitizenRoles(), role)` here

Reply:
👍 done